### ISSUE TITLE
Add Windows script for generating SSL certs needed for SAML2 login flow

### ISSUE THREAD

**Issue by humphd** [Seneca-CDOT/telescope#451]
In #448 and #449 we're working on getting the SAML2 login flow working, and it will require SSL certs for development and testing.  We have a script to do it on *nix, but not Windows.

There are a number of ways this can be done, see https://medium.com/the-new-control-plane/generating-self-signed-certificates-on-windows-7812a600c2d8.  This fix needs to do at least 2 things:

- add a script to `tools/` that can be run in a Windows shell and will generate these certs
- add a line to the Windows install portion of the `.travis.yml` file to run the script in CI

**Comment by manekenpix** [comment]
@Grommers00 I found [Microsoft documentation](https://docs.microsoft.com/en-us/powershell/module/pkiclient/new-selfsignedcertificate?view=win10-ps) with multiples examples of how to create a certificate depending on your needs. I created a certificate using the first example, but since I'm not really sure what the certificate is for I couldn't test it with telescope. Let me know if it helps or if there's anything I could do to test it.
